addons/website_slides/static/src/xml/slide_quiz.xml

```diff
@@ -119,6 +119,15 @@
                     </button>
                 </div>
             </div>
+            <div class="mt-4 o_wslides_js_lesson_quiz_resourceinfo d-none">
+                <div t-if="widget.quiz.slideResources">
+                    <span t-if="widget.quiz.slideResources.length > 0">Need help? Review related content:</span> 
```

Comment:
Checking twice the same thing?

Reply:
true, before we needed to make sure that slideResource existed before we calculated length, but now since we already make sure of that in js we don't really need the first part :D

Reply:
You can check that it is defined and its length is > 1 in the same t-if. 
And you could even try the syntax `widget.quiz.slideResources?.length > 0` to do it.